### PR TITLE
Updating help text when validate cannot find a bin

### PR DESCRIPTION
**Pull request by andrewwippler**
Adding additional help text to help new users configure their librenms install. In my install, fping was not at the expected path. This help text would have helped me.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: validate.php [hunk 1]
```diff
@@ -219,6 +219,8 @@
     $cmd = rtrim(shell_exec("which {$config[$bin]} 2>/dev/null"));
     if (!$cmd) {
         print_fail("$bin location is incorrect or bin not installed");
+        print_fail("You can also manually set the path to $bin by placing the following in config.php:");
+        print_fail("\$config['$bin'] = \"/path/to/$bin\";");
```
Comment (laf) [inline validate.php]: This shouldn't be wrapped in a new print_fail as it show as a new line, you will need to include it in the line above.

Comment (andrewwippler) [inline validate.php]: Ok, should I include line breaks?

Comment (laf) [inline validate.php]: See how it's done here https://github.com/andrewwippler/librenms/blob/4d3120b74a682de7748bc06188e19e85aa79e4f5/validate.php#L147

Comment (andrewwippler) [inline validate.php]: Disregard this next pull. I think you want me to add all missing $bin into an array and then format output.... it that correct?

File: validate.php [hunk 2]
```diff
@@ -207,7 +207,7 @@
 if ($space_check < 512 && $space_check > 1) {
     print_warn('Disk space where '.$config['install_dir'].' is located is less than 512Mb');
 }
-
+d
```
Comment (laf) [inline validate.php]: Think this was added by accident :)

Comment (andrewwippler) [inline validate.php]: It was :)

**Pull request merged**